glommio/src/executor/mod.rs

```diff
@@ -1130,6 +1139,22 @@ impl LocalExecutor {
     /// assert_eq!(res, 6);
     /// ```
     pub fn run<T>(&self, future: impl Future<Output = T>) -> T {
+        let queues = self
+            .queues
+            .borrow()
+            .available_executors
+            .iter()
+            .map(|x| {
+                (
+                    *x.0,
+                    x.1.borrow().name.clone(),
+                    x.1.borrow().shares.clone(),
+                    x.1.borrow().io_requirements,
+                )
+            })
+            .collect::<Vec<_>>();
+        tracing::trace!(id = self.id, ?queues, "LocalExecutor running");
```

Comment:
Why do you feel the queues are important here ?
It seems to me that when the executor starts running this will almost always have the same value, and a predictable one.

Reply:
I don't necessarily.. I admit I hastily added the tracepoints so I've gone ahead and removed most of them in my most recent patch. I really just wanted the dependency to land so people start adding stuff themselves as needed.

Reply:
Agree with the intention, but let's make sure the ones we do add make sense.

In this case it's fair to trace that the executor was created. But the queues are not really useful

Reply:
K, so everything as is but don't log the queues there? I'll go ahead and make that patch.. Again my major goal is to add the infrastructure. Not to mention it allows adding (temporary) trace lines instead of printlns when debugging. Which is nice.

Reply:
that's right.
That we created an executor is good tracing info (adds information), which queues it has at startup is not